src/content/development/code-review/_index.en.md

```diff
@@ -89,6 +89,20 @@ ready to merge (has [required reviews](#two-non-author-committer-approvals-requi
 other relevant code has changed, the new job results don't tell us anything new. Since changes are constantly being merged, it's likely
 another merge commit and set of jobs will be necessary right before merging anyway.
 
+### Mark work-in-progress PRs as drafts
+
+To clearly indicate a Pull Request is still under development, not yet ready for review, mark it as a draft. It is not necessary to modify
+PR summaries or commit messages ("WIP" or "DO NOT MERGE"). Keeping the same PR summary keeps email notifications threaded, and using the
+commit message you plan to merge will allow gitlint to verify it. PRs should typically be marked as drafts if any of the required CI is
+failing.
```

Comment:
I'm not clear on the last sentence. If I read it correctly, if a CI job fails that prevents merging we should change it to a draft even if it's ready for review?

Reply:
I'll try to reword it to be more clear.